examples/ssu-defaults.ini

```diff
@@ -8,7 +8,7 @@
 # Syntax cmd-remove=<old-key1>, <old-key2>, ...
 
 [General]
-configVersion=2
```

Comment:
Hmm.. this is an example etc, but currently installed files are at configVersion=14. Does the delta vs this file matter?

Reply:
The defaults should be synced with all the variables previously added its quite out of date now.

Reply:
TThis is an example and does not have relation to the version on device. But yes, it could use an update to reflect the current state better. Same probably goes for documentation and unit tests and other things.